app/controllers/completions_cancellation_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class CompletionsCancellationController < ApplicationController
+  before_action :confirm_two_factor_authenticated
+
+  def show
+    analytics.exit_to_sp_confirmation_page_visited
```

Comment:
Per prior discussion, I think we should call this:

```suggestion
    analytics.completions_cancellation_visited
```